ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/DistributingExecutor.java

```diff
@@ -61,14 +62,15 @@
 
   public DistributingExecutor(
       final KsqlConfig ksqlConfig,
-      final CommandQueue commandQueue,
+      final CommandRunner commandRunner,
```

Comment:
can we abstract this out as a `Supplier<Boolean>` instead?

Reply:
done